dawn/src/dawn/SIR/SIR.h

```diff
@@ -310,6 +310,9 @@ class FieldDimensions {
   bool isVertical() const { return !horizontalFieldDimension_.has_value(); }
   std::string toString() const;
 
+  // returns number of dimensions (1-3)
+  int numDimensions() const;
```

Comment:
I'd suggest also here a change of name to avoid potential misunderstandings: `numSpatialDimensions()`, do you agree?